01-basics/01-CounterButtonApp/script.js

```diff
@@ -1,3 +1,23 @@
-// import { createApp } from './vendor/vue.esm-browser.js';
+import { createApp, defineComponent } from './vendor/vue.esm-browser.js';
 
-// Создайте Vue приложение
+ // Создайте Vue приложение
+const RootComponent = defineComponent({
+	name: 'Root',
+	vue: {
+		config: {
+			productionTip: false,
+			devtools: true,
+		}
+	},
```

Comment:
У компонентов нет опции vue. Конфигурация приложения настраивается через свойство `config` у приложения, созданного через `createApp`.

Свойства `productionTip` во Vue 3 нет вообще, его убрали. `devtools` - кажется, тоже.

Reply:
Я не знал, как побороть сообщение от плагина Vue "**Vue.js is detected on this page.**
Devtools inspection is not available because it's in production mode or explicitly disabled by the author."

Reply:
![image](https://user-images.githubusercontent.com/94059865/142834048-fd621bea-3396-41a2-a25f-6d9a9cda6f6c.png)

Reply:
Бывает, что просто глючит, и нужно обновить страницу. Используете новые vue-devtools, с поддержкой Vue 3?

Reply:
Он у меня 6-й версии, beta.

Reply:
![image](https://user-images.githubusercontent.com/94059865/142837560-21041768-bc19-4feb-852f-9010d8bf8f97.png)

Reply:
Проблема решилась после добавления опций из этого комментария?

И появляется снова после их удаления?

Reply:
> У компонентов нет опции vue. Конфигурация приложения настраивается через свойство `config` у приложения, созданного через `createApp`.
> 
> Свойства `productionTip` во Vue 3 нет вообще, его убрали. `devtools` - кажется, тоже.

Опцию vue я убрал из описания RootComponent. Но это на кнопку devtools не повлияло. А что и куда добавить, я ещё пока не понял. Что-то типа app.config...? Но что там далее?
![image](https://user-images.githubusercontent.com/94059865/142864777-a83b9414-fee1-4fa4-86de-ba9d2c5d0909.png)

Reply:
Всё должно работать из коробки...

У нас на скрине на вкладках девтулсов есть вкладка Vue. По ней не открываются Vue devtools?

Reply:
Да-да, по ней открываются. Но я открываю её по F12 и потом уже перехожу на вкладку. А по кнопке плагина не открывается. Пишет вон то сообщение, которое на скринах выше. Не критично, в общем-то. Просто интересно, почему так.
![image](https://user-images.githubusercontent.com/94059865/142884613-2a5d4ee7-1ef3-4440-baf2-45fd7fdf2252.png)